source/docs/beta/beta-getting-started/accessing-the-beta-project.rst

```diff
@@ -0,0 +1,36 @@
+Accessing the Beta Project
+==========================
+
+.. note::
+   Only accepted Beta Test teams will have access to the Beta project.
+
+The members identified by your team to communicate with FIRST technical staff will have access to the Beta Test project on the `<https://usfirst.collab.net>`__ site.
+
+Signing in
+----------
+
+.. image:: images/accessing-the-beta-project/login.png
+
+Open your web browser and browse to `<https://usfirst.collab.net>`__. Near the top left corner of the page enter the username and password provided in the Beta test email you received, then click "Log In".
+
+Opening the Beta Project
+------------------------
+
+.. image:: images/accessing-the-beta-project/project-list.png
+
+Your user account should already be added to the Beta Project. To view the project:
+
+   1. Click the Projects tab on the top navigation ribbon.
+   2. Then click on FRC20**Beta in the projects list (image of 2015 as an example).
+
+The Project Homepage
+--------------------
+
+.. image:: images/accessing-the-beta-project/project-home.png
+
+The project homepage contains some statistics and Project News. Throughout the beta, the Project News section may be updated with the latest information from the FIRST technical staff. The top ribbon contains tabs allowing you to navigate to the different sections of the project:
+
+   1. Trackers - The Trackers tab is where the bug tracker for the project is located.
```

Comment:
Any reason for the prepending space?

```suggestion
1. Trackers - The Trackers tab is where the bug tracker for the project is located.
```

Reply:
Nope, I don't mind either way but we should be specific in the styleguide. While lists are not an explicit directive, they are a bit of an odd special case where they need that new line to behave properly.

Reply:
This is how it's done in the Sphinx examples